GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -206,11 +206,22 @@ private async Task LoadSortedRefsAsync()
             _refsOrderDict = null;
 
             await TaskScheduler.Default.SwitchTo();
-            var refsOrderDict = ToDictionary(Module.GetSortedRefs());
+            try
+            {
+                var refsOrderDict = ToDictionary(Module.GetSortedRefs());
+
+                await this.SwitchToMainThreadAsync();
+                _refsOrderDict = refsOrderDict;
+                UpdateRevisionInfo();
+            }
+            catch (GitModule.RefsWarningException ex)
+            {
+                await this.SwitchToMainThreadAsync();
+                MessageBox.Show(this, new TranslationString("The repository refs seem to be broken:").Text + Environment.NewLine + Environment.NewLine + ex.Message,
```

Comment:
⚠️ Please move the translation string to the top with the rest of definitions.
Also please don't use `+` to concat strings, at least use string interpolation.

Reply:
Adding static TranslationStrings for this message and title that will be displayed hardly ever is heavy-handed in my opinion.